### DISCUSSION TITLE
Thank you

### DISCUSSION THREAD

**Discussion by Mistiie** [Monika-After-Story/MonikaModDev#8865] [appreciation]
I've left issues, and every single time, someone jas come to help me. Even when I'm slow and really show no knowledge of computers. Thank you Devs for taking your time to help people, along with the others who come across the problems. But thank you Devs for creating this amazing mod- though you can't do everything on it, it's just so nice to open your computer to a kind Monika. You really put effort into your updates and making the experiences great for everyone. Thank you!

**Comment by ThePotatoGuy** [comment]
[closing as appreciated]
